works/nf2024.html

```diff
@@ -108,102 +87,64 @@ <h2>デモ</h2>
+            <h3 class="h3-left-line">お絵描き予測AI</h3>
+            <p><img src="../assets/nf2024/draw_forecast.png" class="img-fluid"></p>
             <p>
-                *本デモは、StyleGAN2で生成された画像かどうかを判別するのに特化したモデルです。Diffusion Modelなどによる生成画像はこのAIでは判別できません。
+                Quick, Draw！というGoogleのウェブアプリで集められたデータセットを用いて、線画を生成するモデルを学習させました。
+                人間が書いた途中までの絵から、完成形を予想し続きを描いてくれるAIです。
             </p>
             <p>
-                <a href="https://fake-face-image-detection.streamlit.app/" class="btn btn-primary btn-md" target="_blank"><i class="fa-solid fa-arrow-right"></i> フェイク画像判定を試す</a><br>
-                <a href="https://github.com/kyoto-kaira/fake-face-image-detection"><i class="fa-brands fa-github fa-lg"></i></a>
+                <a href="https://draw-forecast-ai.streamlit.app/" class="btn btn-primary btn-md" target="_blank"><i class="fa-solid fa-arrow-right"></i> お絵描きAIを試す</a>
+                <a href="https://github.com/kyoto-kaira/draw-forecast-ai"><i class="fa-brands fa-github fa-lg"></i></a>
             </p>
 
-            <h3 class="h3-left-line">歌詞からアーティスト判定XAI</h3>
-            <img src="https://camo.qiitausercontent.com/ed31060c87d7585a40a24b05c606a898319aa20e/68747470733a2f2f71696974612d696d6167652d73746f72652e73332e61702d6e6f727468656173742d312e616d617a6f6e6177732e636f6d2f302f313930303539392f61313865313633372d636263632d343062322d363537372d3466366264363832383264632e706e67" class="img-fluid">
-            <img src="https://camo.qiitausercontent.com/9fe6a557ba78084e641936900563a9dcd31615f6/68747470733a2f2f71696974612d696d6167652d73746f72652e73332e61702d6e6f727468656173742d312e616d617a6f6e6177732e636f6d2f302f313930303539392f33376331356366362d333633342d323265352d396134302d6338663132346565363964622e706e67" class="img-fluid">
+            <h3 class="h3-left-line">手書きフォント生成AI</h3>
+            <p><img src="../assets/nf2024/personal_font.png" class="img-fluid"></p>
             <p>
-                <b>歌詞を入力すると、どのアーティストの歌詞っぽいかを判定する</b>モデルを開発しました。
-                テキストを文字単位で分割したデータを入力とする
-                「CharacterCNN（→<a href="https://arxiv.org/abs/1509.01626">論文</a>）」
-                で学習を行いました。
+                一部のひらがなの書き方から、その人の筆跡を学習し、筆跡を真似たひらがなを生成するAIです。
             </p>
             <p>
-                さらにこのデモでは、<b>歌詞のどの部分が予測に寄与しているかを
-                「LIME（→<a href="https://arxiv.org/abs/1602.04938">論文</a>）」
-                という説明可能AI（XAI）手法を使って可視化</b>してみました。
-                深層学習による予測はブラックボックスであることが多いですが、このようなXAI手法を使うことで、予測の根拠を可視化することもできます。
+                <a href="https://huggingface.co/spaces/kyoto-kaira/personalized-font-ai" class="btn btn-primary btn-md" target="_blank"><i class="fa-solid fa-arrow-right"></i> 手書きフォント生成AIを試す</a><br>
+                <a href="https://github.com/kyoto-kaira/personalized-font-ai"><i class="fa-brands fa-github fa-lg"></i></a>
             </p>
+            <h3 class="h3-left-line">最強じゃんけんAI</h3>
+            <p><img src="../assets/nf2023/janken.png" class="img-fluid"></p>
```

Comment:
パスが間違っていて画像が表示されてないです

```suggestion
            <p><img src="../assets/nf2024/janken.png" class="img-fluid"></p>
```